gear-programs/vara-tokenizer/tests/gtest.rs

```diff
@@ -0,0 +1,177 @@
+use sails_rs::{calls::*, gtest::calls::*, prelude::*};
```

Comment:
do we really need to have both tests for gclient and gtest? To tell the truth I personally prefer gclient only

Reply:
But the `gclient` tests require a running node and a lot of time to run.
So they marked as `ignored` by default.
I think we should have both.